extensions/resource-deployment/src/test/services/platformService.test.ts

```diff
@@ -0,0 +1,343 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import * as events from 'events';
+import * as fs from 'fs';
+import 'mocha';
+import * as os from 'os';
+import * as cp from 'promisify-child-process';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import { Readable } from 'stream';
+import * as sudo from 'sudo-prompt';
+import * as TypeMoq from 'typemoq';
+import * as vscode from 'vscode';
+import { tryExecuteAction } from '../../common/utils';
+import { OsDistribution } from '../../interfaces';
+import { extensionOutputChannel, PlatformService } from '../../services/platformService';
+
+class TestChildProcessPromise<T> implements cp.ChildProcessPromise {
```

Comment:
This seems like it should probably go in a general stubs.ts (or whatever) module

Reply:
Done.